src/definitions/modules/sidebar.less

```diff
@@ -112,6 +112,7 @@ body.pushable {
 /* Page Context */
 .pushable:not(body) {
   transform: translate3d(0, 0, 0);
+  overflow-y: hidden;
```

Comment:
I am afraid that vertical scroll bar may not appear on page context, when the page context is very long.
(Perhaps my overthinking ?)

If the height of the preview areas is shrinked on https://jsfiddle.net/lubber/vd2ecqgL/9/,  its content can not be fully-scrollable by vertical scroll bar. 
![image](https://user-images.githubusercontent.com/127635/89014477-81fe6a00-d350-11ea-838b-3574bf707b05.png)

Reply:
The /9/ link is the broken one ;)
But yes, i think this is intented to use the body scrollbar whenever the content of the pusher (by design the content needs to be inside the pusher, not as direct child of the context
https://jsfiddle.net/lubber/vd2ecqgL/81/

I think, your example screenshot is due to the jsfiddle console toolbar (i always disable that, because its more "real world" to use the native console

The shrinked example is this one and this is how user should use own overflow areas anyway
https://jsfiddle.net/lubber/vd2ecqgL/77/

If i am still not getting your point, please adjust one of my fiddles accordingly 🙂

Reply:
> due to the jsfiddle console toolbar

Ah, I see.
After I disabled the console, vertical scrollbar seems fine.